components/org.wso2.micro.integrator.coordination/src/main/java/org/wso2/micro/integrator/coordination/util/RDBMSConstantUtils.java

```diff
@@ -47,8 +47,8 @@
     public static final String LOCAL_GROUP_ID = "localGroupId";
     public static final String SCHEDULED_PERIOD = "scheduledPeriod";
 
-    public static final int DEFAULT_HEART_BEAT_INTERVAL = 5000;
-    public static final int DEFAULT_HEART_BEAT_MAX_RETRY = 3;
+    public static final int DEFAULT_HEART_BEAT_INTERVAL = 2000;
```

Comment:
We need to test this thoroughly, This means that the heartbeat operation cannot exceed 2 seconds.